Description: Install Makefile in /usr/share/man instead of /usr/man.
Last-Update: 2015-01-24

--- a/Makefile
+++ b/Makefile
@@ -83,8 +83,8 @@
 		echo $(PREFIX) > install.prefix ; \
 		$(INSTALLDIR) $(PREFIX)/bin ; \
 		$(INSTALLBIN) wmget $(PREFIX)/bin/wmget ; \
-		$(INSTALLDIR) $(PREFIX)/man/man1 ; \
-		$(INSTALLMAN) wmget.1 $(PREFIX)/man/man1/wmget.1 ; \
+		$(INSTALLDIR) $(PREFIX)/share/man/man1 ; \
+		$(INSTALLMAN) wmget.1 $(PREFIX)/share/man/man1/wmget.1 ; \
 
 uninstall:
 		cd `cat install.prefix` && rm -f bin/wmget man/man1/wmget.1
